infrastructure/aws/production/ecs-task-definition-web.json

```diff
@@ -154,6 +154,10 @@
                 {
                     "name": "TASK_RUN_METHOD",
                     "value": "TASK_PROCESSOR"
+                },
+                {
+                    "name": "SSE_SERVER_BASE_URL",
```

Comment:
I think perhaps we should have a separate PR to enable it in production so that we're not holding things up if it doesn't work in staging?